portfolio/src/main/webapp/script.js

```diff
@@ -110,3 +111,12 @@ async function deleteAll() {
   fetch("/delete-data", { method: "POST" });
   window.location.reload();
 }
+
+async function loginStatus(){
+  const response = await fetch("/login");
+  const login= await response.text();
+  console.log(login);
+  if(login.includes("true")){
+    document.getElementById("login").style.display = "block";
+  }   
```

Comment:
Your server is returning HTML snippets but you're not displaying them anywhere. Add the server response to the page so the user gets a login link.